src/components/select-box/template.njk

```diff
@@ -0,0 +1,10 @@
+{% from "label/macro.njk" import govukLabel %}
+
+{% if hasLabelWithText %}
+{{ govukLabel(labelClasses, hasLabelWithText) }}
+{% endif %}
+<select class="govuk-c-select-box {{classes}}" id="{{id}}" name="{{name}}">
```

Comment:
Can you please add spaces before and after the variable names, for consistency with the rest.